xclim/indices/stats.py

```diff
@@ -295,7 +301,11 @@ def func(x):
 
 
 def fa(
```

Comment:
I realize this is a newish function, but I'd mark this as a breaking change in the documentation.

Reply:
What is breaking ?

Reply:
I suppose "breaking" is the wrong word, but the call signature has changed.

Reply:
Indeed, but the functions remain backward compatible, and nothing should break after that change.

Reply:
Fair enough. I would mark this as an internal change in the chnagelog then. Otherwise merge when ready!